app/scripts/modules/core/src/account/AccountTag.tsx

```diff
@@ -12,13 +13,39 @@ export interface IAccountTagState {
 
 @BindAll()
 export class AccountTag extends React.Component<IAccountTagProps, IAccountTagState> {
+  private static cache: {
+    [account: string]: boolean|IPromise<boolean>;
+  } = {};
+
   public state = { isProdAccount: false };
+  private unmounted = false;
```

Comment:
We've used `mounted` in a couple of other places, would be nice to standardize on that.

Reply:
Would be nice to use cancellable Observables instead of having to rely on a variable like this...

Reply:
oh geez. just let me merge this fix

Reply:
merge it, see if anyone cares